test/out.html

```diff
@@ -11,7 +11,7 @@
                 <h1 class="cool nice wow rp__font-size__--COLON12px rp__padding__--COLON8px">
                   Meow
                 </h1>
-                <h2 class="rp__font-size__--COLON12px rp__background__--COLON__--OCTOTHORPF00 rp__padding__--COLON8px">
+                <h2 class="rp__color__--COLON__--OCTOTHORP00F rp__font-size__--COLON12px rp__background__--COLON__--OCTOTHORPF00 rp__padding__--COLON8px">
```

Comment:
Need to fix the classMap generation so this occurs
```suggestion
                <h2 class="rp__color__--COLON__--OCTOTHORP00F rp__color__--COLON__--OCTOTHORP00F___-HOVER rp__font-size__--COLON12px rp__background__--COLON__--OCTOTHORPF00 rp__padding__--COLON8px">
```